### PR TITLE
feat: add and configure 404 page

### PR DESCRIPTION
**Pull request by abayo-luc**
## What does this PR do?
- [x] Configure not found page 
- [x] Add 404 illustration (to be replaced in the future). 

Screenshot 2023-10-10 at 09 41 56

### CODE REVIEW

File: frontend/src/errorPage/NotFound.tsx
Comment (ApostolosBenisis) [inline frontend/src/errorPage/NotFound.tsx:1]: Add a simple render test
Add a story in story book ( stories are they way to test for accessibility issues)

File: frontend/public/404.svg
Comment (ApostolosBenisis) [inline frontend/public/404.svg:1]: where was this  sourced from and what is the license of this image?

**Review by ApostolosBenisis — CHANGES_REQUESTED** [review]

File: frontend/src/errorPage/NotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <Box
+      flex={1}
+      justifyContent="center"
+      alignItems="center"
+      flexDirection="column"
+      display="flex"
+      data-testid={DATA_TEST_ID.NOT_FOUND_CONTAINER}
+    >
+      <img src="/TabiyaLogo.svg" alt="not found" width="250px" data-testid={DATA_TEST_ID.NOT_FOUND_ILLUSTRATION} />
```
Comment (rav3n11) [inline frontend/src/errorPage/NotFound.tsx:21]: I thought we used rems instead of px?

**Review by rav3n11 — APPROVED** [review]

File: frontend/src/errorPage/NotFound.tsx (hunk 1 above)
Comment (ApostolosBenisis) [reply frontend/src/errorPage/NotFound.tsx:21]: we use rem when we want something to have a size be relative to the font.
Here the logo's size is relative to the screen. Additionally this icon here is "special" as it is the main element in the content. Giving it a fixed size (and later adjusting that to be responsive) is fine.

**Pull request merged**